src/endpoint/RestApi.js

```diff
@@ -0,0 +1,45 @@
+import HttpMethod from './HttpMethod.js';
+import env from '../env.js';
+
+const _baseURL = env.baseURL;
+
+
+const request = async (uri, method, body = undefined, timeout = 5000) => {
+  const config = { method };
+  const url = _baseURL + uri;
+
+  if (body) {
+    config.body = JSON.stringify(body);
+    config.headers = { 'Content-Type': 'application/json' };
+  }
+
+  let id = -1;
+  const race = await Promise.race([
+    new Promise(res => id = window.setTimeout(() => res(), timeout)),
```

Comment:
```suggestion
    new Promise(res => {
      id = setTimeout(() => res(), timeout);
    }),
```
아예 줄바꿈을 해주는게 가독성에 더 좋을 것 같아요!